config/config.go

```diff
@@ -50,6 +50,7 @@ func NewConfiguration(clusterName string) Interface {
 
 func (c *Dumper) Dump(clusterfile string) error {
 	if clusterfile == "" {
+		logger.Warn("clusterfile is empty!")
```

Comment:
If using run command clusterfile is "", but it normal, So I think using debug log here is better.

Reply:
OK